wafer/sponsors/templates/wafer.sponsors/sponsors.html

```diff
@@ -6,16 +6,21 @@
 <div class="wafer list">
     {% for sponsor in sponsor_list %}
     <section class="wafer wafer-sponsor">
-    <div>
-        <h2>
-           {% if sponsor.url %}
-           <a href="{{ sponsor.url }}">{{ sponsor.name }}</a>
-           {% else %}
-           {{ sponsor.name }}
-           {% endif %}
-        </h2>
-        {{ sponsor.description.rendered|safe }}
+    <div class="well">
+      <h2 class="bs-callout bs-callout-default bs-callout-{{ sponsor.packages.first.name.lower }}">
+         {% if sponsor.packages.first.name.lower == 'platinum' %}
```

Comment:
We've added a order attribute to sponsor packages, so we could potentially use that. It's still rather hacky, but easily controllable via the site admin.

It might be worthwhile adding the character to display as a field to the package model, for more flexibility?